tests/test_all.py

```diff
@@ -5,6 +5,12 @@
 
 base_dir = os.path.dirname(os.path.realpath(__file__))
 
+
+def _run_unittest_script(script_name, *args):
+    path = os.path.join(base_dir, script_name)
+    run_unittest_script(path, *args)
```

Comment:
This function name doesn't describe what the function does in a way that differentiates it from `run_unittest_script`, can you name it something like `run_unittest_script_in_base_dir`, or simply move the path append logic into `run_unittest_script` if this always gets used?

Reply:
Good thought -- I moved the path append logic into `run_unittest_script` itself.